mitodl/frontend-component-footer-mitol
# cherry pick custom footer link changes

Description: This branch cherry-picks all the changes we made in [`frontend-component-footer`](https://github.com/openedx/frontend-component-footer). Here are the commit hashes:

7e178079d929afec7fb53ff4b59876de9f672fd6
dcb2b114dcabc24b0f819ed606c57270aae5c704
2ed2992d0f220c513933a1f7c4f08b2f93c3a6fe
64a1e58bf9aa5244c9225608d23ce78aad677813
4c8bfb0168483b5eacba23640f40905f71ebacde
ecae64879c34b1c290276b540249fb3d7639f419
c18a4d83e708bc29495941e3e1975ec1fed1b218
863022de2461565eff16fc360b8c88bbb2a8b6ac
2e1194868ffe9d1a1c33255fe0d2f23ff9ff0b7c
3334579aae19b9a0292a23ece595a8a8e9cd4cbc
acfe0d3b65279bcd840542740427d6de6f5d5d72
b7967b49dbfed9b6ff13adbb97250ab907c0a6e0
809c39f8b7890f7c113dbff6328e4dc844a3bdf7
bc5d6e10db1b27849563ff1455deb99631b1632b
6a4c56a93b0d6244eaef4c14b5ce371deca6d82e


Testing Guidelines:

1. Follow the [readme](https://github.com/mitodl/frontend-component-footer-mitol) to configure the footer with the Learning MFE
2. Configure the Footer links in .env.development of the Learning MFE and restart the MFE
3. Links should be visible in footer.
